### DISCUSSION TITLE
Are you adding compatibility for other storage mods?

### DISCUSSION THREAD

**Discussion by zombiethoughts** [xiaocihua/stack-to-nearby-chests#30] [Q&A] [answered]
I noticed you had support for the charm mod by default. I wasn't sure if you were going to support more mods or not, but I figured I'd throw these up here just in case. Support for iron chests restocked / expanded storage / compact storage.

They seem to all be working great. I'm using the most current version of these and your mod as of Mar 09 2024 for 1.19.2.

It should save people some time looking it up themselves. Cheers!

"ironchests:copper_chest",
"ironchests:iron_chest",
"ironchests:gold_chest",
"ironchests:diamond_chest",

"expandedstorage:wood_chest",
"expandedstorage:pumpkin_chest",
"expandedstorage:present",
"expandedstorage:bamboo_chest",
"expandedstorage:moss_chest",
"expandedstorage:iron_chest",
"expandedstorage:gold_chest",
"expandedstorage:diamond_chest",
"expandedstorage:obsidian_chest",
"expandedstorage:netherite_chest",
"expandedstorage:wood_chest_minecart",
"expandedstorage:pumpkin_chest_minecart",
"expandedstorage:present_minecart",
"expandedstorage:bamboo_chest_minecart",
"expandedstorage:moss_chest_minecart",
"expandedstorage:iron_chest_minecart",
"expandedstorage:gold_chest_minecart",
"expandedstorage:diamond_chest_minecart",
"expandedstorage:obsidian_chest_minecart",
"expandedstorage:netherite_chest_minecart",
"expandedstorage:old_wood_chest",
"expandedstorage:old_iron_chest",
"expandedstorage:old_gold_chest",
"expandedstorage:old_diamond_chest",
"expandedstorage:old_obsidian_chest",
"expandedstorage:old_netherite_chest",
"expandedstorage:copper_barrel",
"expandedstorage:exposed_copper_barrel",
"expandedstorage:weathered_copper_barrel",
"expandedstorage:oxidized_copper_barrel",
"expandedstorage:waxed_copper_barrel",
"expandedstorage:waxed_exposed_copper_barrel",
"expandedstorage:waxed_weathered_copper_barrel",
"expandedstorage:waxed_oxidized_copper_barrel",
"expandedstorage:iron_barrel",
"expandedstorage:gold_barrel",
"expandedstorage:diamond_barrel",
"expandedstorage:obsidian_barrel",
"expandedstorage:netherite_barrel",
"expandedstorage:vanilla_wood_mini_chest",
"expandedstorage:wood_mini_chest",
"expandedstorage:pumpkin_mini_chest",
"expandedstorage:red_mini_present",
"expandedstorage:white_mini_present",
"expandedstorage:candy_cane_mini_present",
"expandedstorage:green_mini_present",
"expandedstorage:lavender_mini_present",
"expandedstorage:pink_amethyst_mini_present",
"expandedstorage:iron_mini_chest",
"expandedstorage:gold_mini_chest",
"expandedstorage:diamond_mini_chest",
"expandedstorage:obsidian_mini_chest",
"expandedstorage:netherite_mini_chest",
"expandedstorage:copper_mini_barrel",
"expandedstorage:exposed_copper_mini_barrel",
"expandedstorage:weathered_copper_mini_barrel",
"expandedstorage:oxidized_copper_mini_barrel",
"expandedstorage:waxed_copper_mini_barrel",
"expandedstorage:waxed_exposed_copper_mini_barrel",
"expandedstorage:waxed_weathered_copper_mini_barrel",
"expandedstorage:waxed_oxidized_copper_mini_barrel",
"expandedstorage:mini_barrel",
"expandedstorage:iron_mini_barrel",
"expandedstorage:gold_mini_barrel",
"expandedstorage:diamond_mini_barrel",
"expandedstorage:obsidian_mini_barrel",
"expandedstorage:netherite_mini_barrel",

"compact_storage:compact_chest_white",
"compact_storage:compact_chest_orange",
"compact_storage:compact_chest_magenta",
"compact_storage:compact_chest_light_blue",
"compact_storage:compact_chest_yellow",
"compact_storage:compact_chest_lime",
"compact_storage:compact_chest_pink",
"compact_storage:compact_chest_gray",
"compact_storage:compact_chest_light_gray",
"compact_storage:compact_chest_cyan",
"compact_storage:compact_chest_purple",
"compact_storage:compact_chest_blue",
"compact_storage:compact_chest_brown",
"compact_storage:compact_chest_green",
"compact_storage:compact_chest_red",
"compact_storage:compact_chest_black",
"compact_storage:compact_barrel_white",
"compact_storage:compact_barrel_orange",
"compact_storage:compact_barrel_magenta",
"compact_storage:compact_barrel_light_blue",
"compact_storage:compact_barrel_yellow",
"compact_storage:compact_barrel_lime",
"compact_storage:compact_barrel_pink",
"compact_storage:compact_barrel_gray",
"compact_storage:compact_barrel_light_gray",
"compact_storage:compact_barrel_cyan",
"compact_storage:compact_barrel_purple",
"compact_storage:compact_barrel_blue",
"compact_storage:compact_barrel_brown",
"compact_storage:compact_barrel_green",
"compact_storage:compact_barrel_red",
"compact_storage:compact_barrel_black",

**Comment by xiaocihua** [comment, marked as answer]
Thank you @zombiethoughts !
